### PR TITLE
DNM Just testing together change in the operator and u/s code

### PR DESCRIPTION
**Pull request by slawqo**
DNM This is just test of depends-on on u/s changes

// This one should be merged to make it working always
Depends-On: https://github.com/openstack-k8s-operators/ci-framework/pull/1892

// This is just test change in u/s repo to test that changes in the openstack code and operator can be tested togeter
Depends-On: https://review.opendev.org/c/openstack/neutron/+/922783

Related: [OSP-26653](https://issues.redhat.com//browse/OSP-26653)

### CODE REVIEW

**Comment by danpawlik** [comment]
recheck

File: zuul.d/project.yaml [hunk 1]
```diff
@@ -4,4 +4,9 @@
     github-check:
       jobs:
         - openstack-k8s-operators-content-provider
```
Comment (SeanMooney) [inline zuul.d/project.yaml:6]: openstack-k8s-operators-content-provider shoudl have been removed in this test
as its replaced by openstack-meta-content-provider

Comment (slawqo) [reply zuul.d/project.yaml]: Done

**Comment by karelyatin** [comment]
recheck fresh

**Pull request closed without merging**